colossalai/fx/codegen/activation_checkpoint_codegen.py

```diff
@@ -105,10 +105,10 @@ def _gen_ckpt_usage(label, activation_offload, input_vars, output_vars, use_reen
     """
     outputs = ', '.join(output_vars)
     inputs = ', '.join(input_vars)
-    return f'{outputs} = colossalai.utils.activation_checkpoint.checkpoint(checkpoint_{label}, {activation_offload}, {inputs}, use_reentrant={use_reentrant})'
+    return f'{outputs} = colossalai.utils.activation_checkpoint.checkpoint(checkpoint_{label}, {activation_offload}, self, {inputs}, use_reentrant={use_reentrant})'
```

Comment:
Ideally this should be `checkpoint(self.checkpoint_{label}, ...)`, the current implementation does not bind the checkpoint method to the object.

Reply:
Ohhh yeah my bad